common/changes/@itwin/imodels-client-management/ab-create-version-bump-pipeline_2024-05-29-06-44.json

```diff
@@ -0,0 +1,55 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/imodels-client-management",
+      "comment": "Add `forkedFrom` response property in `CreateIModelOperationDetails`.",
+      "type": "minor"
+    },
+    {
+      "packageName": "@itwin/imodels-client-management",
+      "comment": "Add fork iModel operation.",
+      "type": "minor"
+    },
+    {
+      "packageName": "@itwin/imodels-client-management",
+      "comment": "Add `containersEnabled` property.",
+      "type": "minor"
+    },
+    {
+      "packageName": "@itwin/imodels-client-management",
+      "comment": "Fix consumer provided `timeOutInMs` not being respected in iModel operations.",
+      "type": "minor"
+    },
+    {
+      "packageName": "@itwin/imodels-client-management",
+      "comment": "Add Named Version `$search` query parameter.",
+      "type": "minor"
+    },
+    {
+      "packageName": "@itwin/imodels-client-management",
+      "comment": "Add iModel `$search` query parameter.",
+      "type": "minor"
+    },
+    {
+      "packageName": "@itwin/imodels-client-management",
+      "comment": "Add HTTP status code to iModelsError.",
+      "type": "minor"
+    },
+    {
+      "packageName": "@itwin/imodels-client-management",
+      "comment": "Add Changeset Extended Data operations.",
+      "type": "minor"
+    },
+    {
+      "packageName": "@itwin/imodels-client-management",
+      "comment": "Add ordering by `createdDateTime` for Named Versions.",
+      "type": "minor"
+    },
+    {
+      "packageName": "@itwin/imodels-client-management",
```

Comment:
I think we just released all these changes yesterday. Should we remove these? If they appear as changes made with the next release it will kind of be inaccurate?

Reply:
I'm not sure, we don't have any changelog since 5.0.0. So this would be the changes from 5.0.0 to 5.6.2. Should we just keep it empty?

Reply:
ah I see, let's keep it then